### PR TITLE
Initialize go backend

### PR DESCRIPTION
**Pull request by jotak**
- Move all frontend to web/
- Init go (mod, vendor, makefile, cmd/pkg...)
- Create simple server + static and dummy API routes with mux

### CODE REVIEW

**Comment by jotak** [comment]
(shamelessly borrowing some test code from @jmazzitelli ;-) )

**Review by mariomac — APPROVED** [review]

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+./web/node_modules
+./web/dist
+.git*
```
Comment (jpinsonneau) [inline .dockerignore:3]: is this for `.github/workflows/main.yml` ?

**Review by jpinsonneau — APPROVED** [review]

File: .dockerignore (hunk 1 above)
Comment (jotak) [reply .dockerignore:3]: it's mostly for `.git/` which otherwise takes time to copy as we run `COPY . .`
the wildcard may not be necessary

**Comment by jotak** [comment]
small change, replacing the yaml config with just program arguments as there's just a few of them

**Pull request merged**